backend/src/main/java/com/team4/backend/dto/InternshipOfferDTO.java

```diff
@@ -0,0 +1,49 @@
+package com.team4.backend.dto;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.team4.backend.model.InternshipOffer;
+import com.team4.backend.model.User;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.io.Serializable;
+import java.time.LocalDate;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Data
+@NoArgsConstructor
+public class InternshipOfferDTO implements Serializable {
+
+    private String id;
+
+    @JsonFormat(pattern = "yyyy-MM-dd")
+    private LocalDate limitDateToApply, beginningDate, endingDate;
+
+    private String companyName, emailOfMonitor, description;
+
+    private Float minSalary, maxSalary;
+
+    private List<String> listEmailInterestedStudents, listEmailExclusiveStudents;
```

Comment:
Please use single declaration per line (to keep code consistency with the rest of the code base)